[2024-03-15 14:21:02,088] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-15T14:20:00+00:00 [queued]>
[2024-03-15 14:21:02,111] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-15T14:20:00+00:00 [queued]>
[2024-03-15 14:21:02,111] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 14:21:02,111] {taskinstance.py:1088} INFO - Starting attempt 1 of 6
[2024-03-15 14:21:02,111] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 14:21:02,124] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): retrieve_raw_data> on 2024-03-15T14:20:00+00:00
[2024-03-15 14:21:02,133] {standard_task_runner.py:52} INFO - Started process 5856 to run task
[2024-03-15 14:21:02,137] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'retrieve_raw_data', '2024-03-15T14:20:00+00:00', '--job-id', '1911', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp4xd9rf6h', '--error-file', '/tmp/tmp6a3899o9']
[2024-03-15 14:21:02,137] {standard_task_runner.py:77} INFO - Job 1911: Subtask retrieve_raw_data
[2024-03-15 14:21:02,187] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.retrieve_raw_data 2024-03-15T14:20:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-15 14:21:02,257] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=retrieve_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-15T14:20:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-15T14:20:00+00:00
[2024-03-15 14:21:02,757] {python.py:151} INFO - Done. Returned value was: None
[2024-03-15 14:21:02,768] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=retrieve_raw_data, execution_date=20240315T142000, start_date=20240315T142102, end_date=20240315T142102
[2024-03-15 14:21:02,800] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-03-15 14:21:02,831] {local_task_job.py:151} INFO - Task exited with return code 0
